src/components/UncontrolledTable.js

```diff
@@ -196,6 +196,10 @@ export default class UncontrolledTable extends React.Component {
     }
   }
 
+  componentDidMount() {
+    this.props.onVisibleRowsChange(this.getVisibleRows());
```

Comment:
Can you add a test for this behavior as well? Just in case future changes won't break this

Reply:
Yes! Actually, the tests I wrote for #943 were broken. Fixing that all up now.